WcaOnRails/db/seeds/development/competitions.seeds.rb

```diff
@@ -42,6 +44,11 @@ def random_wca_value
       latitude_degrees: rand(-90.0..90.0),
       longitude_degrees: rand(-180.0..180.0),
     )
+    eventIds.each do |eventId|
+      competition.events << Event.find(eventId)
```

Comment:
Hmm... would it be clear to instead turn `eventIds` into an array of actual `Event` objects (called `events`) and then do something like this:

```ruby
competition.events = events
```